src/jit/emitarm64.cpp

```diff
@@ -2864,6 +2864,8 @@ emitter::code_t emitter::emitInsCode(instruction ins, insFormat fmt)
         case EA_8BYTE:
             result = INS_OPTS_1D;
             break;
+        default:
```

Comment:
maybe you could add some kind of "// TODO-Cleanup: add unreached() here" comments to the "default: break;" cases